### PR TITLE
show the equal portion for each slider value

### PR DESCRIPTION
**Pull request by hueitan**
Phabricator Link: -

### Problem Statement

The visual UI of the text size component doesn't feel right

### Solution

calculate the portion to get the nearest value

### Note

merge to refactor branch

### CODE REVIEW

File: src/components/TextSize.js [hunk 1]
```diff
@@ -9,7 +9,8 @@ let originalTextSize
 export const TextSize = ({ close }) => {
   const i18n = useI18n()
   const { textSize, setTextSize } = useContext(FontContext)
-  const sliderValue = ['0', '16.6', '33.2', '49', '66.6', '83.2', '94']
+  const sliderPortion = 100 / 6
+  const sliderValue = Array.from({ length: 7 }, (v, i) => i * sliderPortion)
```
Comment (hueitan) [inline src/components/TextSize.js:13]: Question here: what about this constant 7 , we get it from the articleTextSize util?

so the constant can be reused for the value of portion and array length.

Comment (medied) [reply src/components/TextSize.js:13]: You mean like having a variable that we can reuse for both creating the `sliderValue` and in `articleTextSize` util? `MAX_SIZE`?

Comment (medied) [reply src/components/TextSize.js:13]: Neat idea by the way, I think it looks a bit smoother

File: style/textsize.less [hunk 2]
```diff
@@ -50,6 +50,7 @@
 				top: -6px;
 				border: solid 3px #fff;
 				background-color: #36c;
+				transform: translateX( -50% );
```
Comment (medied) [inline style/textsize.less:53]: 👌

**Comment by hueitan** [comment]
@medied this is done + direction handling.

File: src/components/TextSize.js [hunk 3]
```diff
@@ -37,20 +40,20 @@ export const TextSize = ({ close }) => {
     center: i18n('softkey-ok'),
     onKeyCenter: close,
     onKeyBackspace,
-    onKeyFixedArrowLeft,
-    onKeyFixedArrowRight
+    onKeyArrowLeft,
+    onKeyArrowRight
```
Comment (medied) [inline src/components/TextSize.js:44]: Oh thanks for updating these!

**Comment by medied** [comment]
> @medied this is done + direction handling.

Awesome, thanks! Will test and merge now then

**Pull request merged**